sample/android/app/build.gradle

```diff
@@ -115,8 +110,8 @@ android {
     }
     release {
       signingConfig signingConfigs.release
-      minifyEnabled enableProguardInReleaseBuilds
-      proguardFiles getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro"
+      minifyEnabled true
+      proguardFiles "proguard-rules.pro"
```

Comment:
Do we want to drop getDefaultProguardFile() part

Reply:
Undoing this change...

Reply:
Done